packages/instrumentation_adapter/example/android/app/src/main/kotlin/com/example/instrumentation_adapter_example/MainActivity.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.instrumentation_adapter_example
+
+import android.os.Bundle
+
+import io.flutter.app.FlutterActivity
+import io.flutter.plugins.GeneratedPluginRegistrant
+
+class MainActivity: FlutterActivity() {
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    GeneratedPluginRegistrant.registerWith(this)
+  }
+}
```

Comment:
Do you intent to mix the use of java and kotlin? I'm fine with either way, but usually keeping it consistent is better to reduce confusion or dependency issues.

Reply:
hmm, I'll guess I'll switch it to the Java version. Flutter create is defaulting to Kotlin but most of the plugins repo is still Java.